system-variables.md

```diff
@@ -1357,8 +1357,10 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Type: Boolean
-- Default value: `ON`
+- Default value: `OFF`
 - This variable controls whether to enable the dynamic memory control feature for the operator that reads data. By default, this operator enables the maximum number of threads that [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) allows to read data. When the memory usage of a single SQL statement exceeds [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) each time, the operator that reads data stops one thread.
+- When the operator that reads data has only one thread and a single SQL statement uses memory that exceeds the value of `tidb_mem_quota_query`, other memory control actions will be triggered, such as [spilling to a disk](/tidb-configuration-file.md#oom-use-tmp-storage).
+- This variable controls memory usage effectively when an SQL statement only reads data. If computing operations (such as join or aggregation operations) are required, memory usage might not be under control of `tidb_mem_quota_query`, which increases the risk of OOM.
```

Comment:
Move L1363 to L1376 or L1368

Reply:
Fixed in [3ab4425](https://github.com/pingcap/docs/pull/10523/commits/3ab4425d426befbc419b99523cc6f186c083f68d)